assembly/benches/sha256.rs

```diff
@@ -0,0 +1,30 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use std::time::Duration;
+use miden_assembly::{self, Assembler};
+
+
+fn script_compilation(c: &mut Criterion) {
+    let mut group = c.benchmark_group("sha256");
+    group.measurement_time(Duration::from_secs(300));
+
+    group.bench_function("Bench the assembly script compilation", |bench| {
```

Comment:
And here, we could probably just use "sha256" as the name of the benchmark.

Reply:
Done!